js/controller.js

```diff
@@ -617,6 +641,26 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       }
     },
 
+    onTouching: function(params, isOnVideoClick) {
+      if (this.state.playerState == CONSTANTS.STATE.PLAYING) {
+        if (this.state.isVideo360 && isOnVideoClick) {
+          this.mb.publish(OO.EVENTS.TOUCHING, this.focusedElement, params);
```

Comment:
Where does params come from? It is not defined from within the scope of this function.

Reply:
It is params which we pass to the function (yaw, roll,  pitch).